orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -47,7 +49,17 @@ class CreateWebhookTask implements RetryableTask {
     def customHeaders = stage.context.customHeaders
     boolean waitForCompletion = (stage.context.waitForCompletion as String)?.toBoolean()
 
-    def response = webhookService.exchange(method, url, payload, customHeaders)
+    def response
+    try {
+       response = webhookService.exchange(method, url, payload, customHeaders)
+    } catch (HttpStatusCodeException e) {
```

Comment:
@jervi - updated to the less specific `HttpStatusCodeException`, which is the superclass of both `HttpClientErrorException` and `HttpServerErrorException`.